client/my-sites/upgrades/index.js

```diff
@@ -284,10 +285,19 @@ module.exports = function() {
 		);
 
 		page(
-			'/checkout/:domain/:plan_name?',
+			'/checkout/:domain/:planName?',
 			adTracking.retarget,
 			controller.siteSelection,
 			upgradesController.checkout
 		);
+
+		page(
+			'/start-trial/:planName/:domain',
```

Comment:
+1, makes sense to use `:site` for site and `:domain` for domain. Would be nice to fix it later as you said